datadotworld/client/api.py

```diff
@@ -190,15 +188,7 @@ def update_dataset(self, dataset_key, **kwargs):
         """
         request = self.__build_dataset_obj(
```

Comment:
`__build_dataset_obj` is never invoked by `create_dataset` ? If not, then can we just remove the 2nd lambda arg from `__build_dataset_obj` entirely? Then you don't need all the dummy lambdas.